### ISSUE TITLE
TypeError using `refreshAuthCookies` with next-firebase-auth-edge 0.8.8 and Next.js 13.5.5

### ISSUE THREAD

**Issue by steve-marmalade** [awinogrodzki/next-firebase-auth-edge#107]
Hello again! I just upgraded to next-firebase-auth-edge 0.8.8 and Next.js 13.5.5 and am getting a TypeError using `refreshAuthCookies`. The error goes away if I downgrade to Next.js 13.5.4.

I have some middleware code that is very similar to the [README example](https://github.com/awinogrodzki/next-firebase-auth-edge#refreshauthcookies-in-middleware)
```typescript
import { NextResponse } from "next/server";

const refreshedResponse = new NextResponse(JSON.stringify(user.customClaims), {
  status: 200,
  headers: { "content-type": "application/json" },
});

await refreshAuthCookies(token, refreshedResponse, commonOptions);
```

```
middleware.ts:51:5 - error TS2345: Argument of type 'import("/home/user/app/node_modules/next/dist/server/web/spec-extension/response").NextResponse<unknown>' is not assignable to parameter of type 'import("/home/user/app/node_modules/next-firebase-auth-edge/node_modules/next/dist/server/web/spec-extension/response").NextResponse<unknown>'.
  Property '[INTERNALS]' is missing in type 'import("/home/user/app/node_modules/next/dist/server/web/spec-extension/response").NextResponse<unknown>' but required in type 'import("/home/user/app/node_modules/next-firebase-auth-edge/node_modules/next/dist/server/web/spec-extension/response").NextResponse<unknown>'.

51     refreshedResponse,
```

**Comment by steve-marmalade** [comment]
FYI this appears to be affecting `NextRequest` as well

**Comment by awinogrodzki** [comment]
Hey @steve-marmalade, thank you so much for reporting!

I will release a fix for this in `0.9.0` today.

**Comment by awinogrodzki** [comment]
@steve-marmalade could you install `next-firebase-auth-edge@0.9.0-1` and verify it works?

**Comment by steve-marmalade** [comment]
I am still seeing this issue in both `0.9.0` and `0.9.0-1` with [v13.5.7-canary.30](https://github.com/vercel/next.js/releases/tag/v13.5.7-canary.30)

**Comment by awinogrodzki** [comment]
It seems that property `'[INTERNALS]'` is changing quite often. I probably need to find a way to exclude it from the `NextResponse` to avoid such conflicts in future. I will try to address that in upcoming PRs

**Comment by awinogrodzki** [comment]
Hey @steve-marmalade,

I am working on compatibility with latest Next.js 13 Canary release. Could you install `next-firebase-auth-edge@0.9.5-1` to see if you still get the error? I think the problem was `next@^13.5.5` in `devDependencies`, but I wasn't able to reproduce the issue in starter

After this is fixed I will move on to compatibility with Next.js 14 :-)

**Comment by awinogrodzki** [comment]
FYI: Next.js 14 support was released in `0.10.0`

**Comment by awinogrodzki** [comment]
I will close the issue now, let me know if there's anything else I can help with
